packages/app/src/messages/helpers/threads.helpers.ts

```diff
@@ -79,15 +80,19 @@ const isContactWithAnyNumber = (
   | (Contact & { primaryPhoneNumber: string })
   | (Contact & { secondaryPhoneNumber: string }) => {
   return (
-    isPhoneNumberValid(contact.primaryPhoneNumber) ||
-    isPhoneNumberValid(contact.secondaryPhoneNumber)
+    isPhoneNumberNotEmpty(contact.primaryPhoneNumber) ||
+    isPhoneNumberNotEmpty(contact.secondaryPhoneNumber)
   )
 }
 
-const isPhoneNumberValid = (phoneNumber = ""): phoneNumber is string => {
+const isPhoneNumberNotEmpty = (phoneNumber = ""): phoneNumber is string => {
   return phoneNumber !== ""
 }
 
+export const isPhoneNumberValid = (phoneNumber: string): boolean => {
```

Comment:
Are you sure this change won't make a regression? Can we catch up 15 minutes before the daily so you can help me understand what it's for? 🙏

Reply:
sure, so 10:30 :)

Reply:
Everything is git. My mistake in code review. I'm sorry! 🙏